src/server/network/protocol/protocollogin.cpp

```diff
@@ -180,6 +180,13 @@ void ProtocolLogin::onRecvFirstMessage(NetworkMessage &msg) {
 		return;
 	}
 
+	// OTCv8 version detection
+	uint16_t otclientV8 = 0;
```

Comment:
Move to protocollogin.h? This is not used here.

Reply:
É verdade, eu queria colocar como veio.